addon-test-support/load.js

```diff
@@ -19,4 +22,69 @@ export default function loadEmberExam() {
   if (window.Testem) {
     patchTestemOutput(TestLoader);
   }
+
+  testLoader = new TestLoader();
 }
+
+// loadTests() is equivalent to ember-qunit's loadTest() except this does not create a new TestLoader instance
+function loadTests() {
+  testLoader.loadModules();
+}
+
+// setupQUnitCallBacks() registers QUnit callbacks neeeded for the load-balance option.
+function setupQUnitCallbacks(qunit) {
+  if (!location.search.includes("_loadBalance")) {
+    return;
+  }
+  qunit.begin(function() {
+    return new Promise(function(resolve) {
+      Testem.once('next-module-response', function getTestModule(moduleName) {
+        if (moduleName !== undefined) {
+          loadIndividualModule(moduleName);
+        }
+
+        // if no tests were added, request the next module
+        if (qunit.config.queue.length === 0) {
+          Testem.emit('get-next-module');
+        } else {
+          Testem.removeEventCallbacks('next-module-response', getTestModule);
+          resolve();
```

Comment:
This resolve may not always be called, if anything in this callback throws, we simply wont invoke this resolve nor do we reject.

Reply:
`loadIndividualModule` is the callback resolves the promises. The function calls `testLoader.loadIndividualModule` that’s in `patch-test-loader` and  `testLoader.loadIndividualModule` requires and unsess a given module. It seems like `require` and `unsee` already handles an error that might be thrown. I’m thinking `loadIndividualModule` won’t cause execution stops or unresolves the promise.
https://github.com/ember-cli/ember-cli-test-loader/blob/master/addon-test-support/index.js#L71

Reply:
Would it at least be prudent to `reject` if errors occur?